notebooks/text_classification/solutions/2_keras_for_text_classification.ipynb

```diff
@@ -910,6 +910,218 @@
     "cnn_model.summary()"
```

Comment:
<p>When running through this lab, the dnn_history consisted of only 1 point. I needed to change PATIENCE=2 to get reasonable plots for train/validation loss. This is new behavior from having run the labs recently, which is odd, but might consider changing PATIENCE=0 to PATIENCE=5.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Reply:
<p>Done!</p>